examples/fair_prep_results.py

```diff
@@ -0,0 +1,141 @@
+pre_processors = [NoPreProcessing(), DIRemover(1.0), DIRemover(0.5), Reweighing()]
+
+path = os.path.join(os.getcwd(), "logs")
+
+def extract_info(present_acc,present_di,present_fnr,present_fpr):
+    '''
+    Function to extract the metrics (accuracy, DI, false_positive_rate, false_negative_rate)value from the log files 
+    '''
+    try:
+        for folderName, subfolders, filenames in os.walk(path):      
+            for subfolder in subfolders:
+                filelist = os.listdir(path+'/'+subfolder)
+                for f in filelist:
+                    file_path = path+'/'+subfolder+'/'+f
+                    with open(file_path,'r+') as f:
+                        for line in f:
+                            
+                            if line.startswith("test,,disparate_impact"):
+                                di = line.split(',')
+                                di_value = di[-1].rstrip("\n")
+                                present_di.append(float(di_value))
+                            
+                            elif line.startswith("test") and 'accuracy' in line and 'None' in line:
+                                acc = line.split(',')
+                                acc_val = acc[-1].rstrip("\n")
+                                present_acc.append(float(acc_val))
+                            
+                            elif line.startswith("test") and 'generalized_false_positive_rate' in line and 'None' in line:
+                                fnr = line.split(',')
+                                fnr_val = fnr[-1].rstrip("\n")
+                                present_fnr.append(float(fnr_val))
+                            
+                            elif line.startswith("test") and 'generalized_true_positive_rate' in line and 'None' in line:
+                                fpr = line.split(',')
+                                fpr_val = fpr[-1].rstrip("\n")
+                                present_fpr.append(float(fpr_val))
+                shutil.rmtree(path+'/'+subfolder)            
+    except:
+        pass
+    
+    return present_acc,present_di,present_fnr,present_fpr
+
+def exp_run_pre_pro_param(seed,learn,pre,post):
+    '''
+    Experiment function to run the experiments
+    '''
+    exp = GermanCreditDatasetSexExperiment(
+    fixed_random_seed=seed,
+    train_data_sampler=CompleteData(),
+    missing_value_handler=CompleteCaseAnalysis(),
+    numeric_attribute_scaler=NamedStandardScaler(),
+    learners=[learn],
+    pre_processors=[pre],
+    post_processors=[post]
+    )
+    
+    exp.run()
+
+#Mergining processors to one format for input
+processors = [[NoPreProcessing(),NoPostProcessing()], [DIRemover(1.0),NoPostProcessing()], [DIRemover(0.5),NoPostProcessing()], [Reweighing(),NoPostProcessing()],
+              [NoPreProcessing(), RejectOptionPostProcessing()],[NoPreProcessing(),CalibratedEqualOddsPostProcessing()]]
+
+
+
+accuracy, disp_imp, fpr, fnr  = [], [], [], []
```

Comment:
please initialize each variable on its own line and give them readable names

Reply:
updated.